ZenPacks/zenoss/HttpMonitor/datasources/HttpMonitorDataSource.py

```diff
@@ -115,7 +115,15 @@ def collect(self, config):
         useSsl = ast.literal_eval(ds0.params['useSsl'])
         url = ds0.params['url']
         ipaddress = ds0.params['ipAddress']
-        onRedirect = ast.literal_eval(ds0.params['onRedirect'])
+        try:
+            onRedirect = ast.literal_eval(ds0.params.get('onRedirect', False))
```

Comment:
`literal_eval` only accepts strings, so the default value in `get` should be `"False"`, not `False`.